missilewars-plugin/src/main/java/de/butzlabben/missilewars/configuration/arena/Arena.java

```diff
@@ -56,43 +55,26 @@ public class Arena implements Cloneable {
     @SerializedName("money") private MoneyConfiguration money = new MoneyConfiguration();
     @SerializedName("equipment_interval") private EquipmentIntervalConfiguration interval = new EquipmentIntervalConfiguration();
     @SerializedName("missile") private MissileConfiguration missileConfiguration = new MissileConfiguration();
-    @SerializedName("area") private FlatArea gameArea = new FlatArea(-30, -72, 30, 72);
-    
+    @Setter @SerializedName("area") private AreaConfiguration areaConfig = new AreaConfiguration(-30, 0, -72, 30, 256, 72);
+
```

Comment:
Now we have a config object (`AreaConfiguration`). From this the `GameArea` is then generated.